rules/review-microservice-boundaries/rule.md

```diff
@@ -0,0 +1,54 @@
+---
+type: rule
```

Comment:
Needs an `seoDescription`. Can you ChatGPT one up?

Reply:
Done